doc/about.rst

```diff
@@ -7,8 +7,8 @@ including
 `HoloViews <http://holoviews.org>`_,
 `GeoViews <http://geo.holoviews.org>`_,
 `hvPlot <https://hvplot.pyviz.org>`_,
-`Datashader <https://github.com/bokeh/datashader>`_,
-`Param <https://github.com/ioam/param>`_, and
+`Datashader <https://datashader.org>`_,
```

Comment:
That datashader link doesn't work for me...maybe it's down at the moment?

Reply:
Looks like it needs to be http, not https.